cluster/local/README.md

```diff
@@ -42,8 +44,34 @@ For complete list of subcommands supported by `minikube.sh`, run:
 cluster/local/minikube.sh
 ```
 
+#### MicroK8s (recommended for Linux)
+Starting the cluster on Minikube is as simple as running:
+```console
+cluster/local/microk8s.sh up
+```
+
+To copy Crossplane image generated from your local build into the MicroK8s container registry, run the following commands after `microk8s.sh up` succeeded:
+```console
+cluster/local/microk8s.sh helm-install
+```
+
+Resetting the MicroK8s cluster can be done with:
+```console
+cluster/local/microk8s.sh clean
+```
+
+Stopping the MicroK8s cluster can be done with:
+```console
+cluster/local/microk8s.sh clean
+```
+
+For complete list of subcommands supported by `microk8s.sh`, run:
+```console
+cluster/local/microk8s.sh
+```
+
 ## Run Tests
-From the project root do the following:
+From the project root do the following (minikube used for example, but same pattern can be followed with any Kubernetes distribution):
```

Comment:
I would reword this.

```
The following sections provide commands helpful for development environments.  
`minikube.sh` may be replaced with [`local.sh`](link-to-above) or [`microk8s.sh`](link-to-above) in those environments.
These commands expect to be run from the project root.
```

Reply:
I don't know that these scripts are actually dependent on being run from the root.

Reply:
I like the rewording. The reason it specifies root is because of the `make` commands and the full paths for the scripts.